py_script/plot_dps.py

```diff
@@ -1,3 +1,7 @@
+"""
+Plots the dps of all weapons on a neat graph. Used to generate graphics for dps_vis.
+"""
+
```

Comment:
if ur being specific it generates data for dps_vis

Reply:
ok